.template-helpers/variant.sh

```diff
@@ -23,9 +24,16 @@ for tag in "${tags[@]}"; do
 	fi
 done
 if [ "$text" ]; then
-	# give a little space
-	echo
-	echo
-	cat "$dir/variant.md"
-	echo "$text"
+  latest=($(curl -sSL $url | sed -e 's/git:\/\/github.com\///' -e 's/@/ /' - | grep "latest"))
+  dockerfile='https://raw.githubusercontent.com/'"${latest[1]}"'/'"${latest[2]}"'/'"${latest[3]}"'/Dockerfile'
```

Comment:
I know there is [one place](https://github.com/infosiftr/docker-library-docs/blob/master/.template-helpers/generate-dockerfile-links-partial.sh#L52) where we have an assumption on it being a github repo. but that may not always be the case.  I just want to make sure we are aware of this possible problem later (especially if [pypy](https://bitbucket.org/pypy/pypy) wants to maintain their image).